lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1239,41 +1290,50 @@ export class UserAgentApplication {
    * @param {string} [hash=window.location.hash] - Hash fragment of Url.
    * @hidden
    */
-  private processCallBack(hash: string): void {
+  private processCallBack(hash: string, stateInfo: ResponseStateInfo, parentCallback?: Function): void {
```

Comment:
need comments update here as well

Reply:
Same here. I did a final check for comments in a separate PR that has prompt validation. Will double check again.